raiden/network/transport/matrix/rtc/web_rtc.py

```diff
@@ -584,9 +583,8 @@ def stop(self) -> None:
             conn.send_hangup_message()
 
         for partner_address, conn in self._address_to_connection.copy().items():
-            task = self.close_connection(partner_address)
+            self.close_connection(partner_address)
```

Comment:
Is there a reason why you don't add the `conn.close()` task the all coroutines? will it ever wait on the `close()` coroutine somewhere?

Reply:
Once the task gets to run, it will wait for all the coroutines, in the new `_close` method.

Reply:
Yes but I mean where do you wait on this task?

Reply:
This is now handled by the new `wait` method on `_RTCConnection`, that waits for all tasks and greenlets.